packages/vendure-plugin-customer-managed-groups/src/customer-managed-groups.service.ts

```diff
@@ -391,7 +423,8 @@ export class CustomerManagedGroupsService {
       !input.title &&
       !input.firstName &&
       !input.lastName &&
-      !input.emailAddress
+      !input.emailAddress &&
+      !(input.addresses && input.addresses.length)
```

Comment:
Minor readability improvement
```suggestion
      !input.addresses?.length
```